client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -2,15 +2,12 @@
  * External dependencies
  */
 import React, { Component, PropTypes } from 'react';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import FoldableCard from 'components/foldable-card';
 import ProductFormDetailsCard from './product-form-details-card';
-import ProductVariationTypesForm from './product-variation-types-form';
-import FormToggle from 'components/forms/form-toggle';
+import ProductFormVariationsCard from './product-form-variations-card';
 
 export default class ProductForm extends Component {
```

Comment:
Yeah, we should probably look into using a function for this if we can. @justinshreve and I will look it over to make sure it is won't gain any more state/complexity and if not, make it a function.